docs/developer-guide/working-with-plugins.md

```diff
@@ -140,6 +140,11 @@ ruleTester.run("custom-plugin-rule", rule, {
 });
 ```
 
+#### Customizing RuleTester
+
+RuleTester internally uses Mocha's `describe` and `it` methods when available to create tests for each valid and invalid case. If you use another test runner, you can override `RuleTester.describe` and `RuleTester.it` to make RuleTester compatible with it and have proper individual tests and feedback.
```

Comment:
Super content: says much in few words. Here are a few suggestions about grammar of the first sentence:
* move to beginning: To create tests for each valid and invalid case,
* is internally implied in this context?
* avoid possessive on product name and include its tag line: RuleTester uses `describe` and `it` methods from the Mocha test framework when it is available.